examples/hello.rs

```diff
@@ -34,6 +37,12 @@ impl eframe::App for MyApp {
             // draws the actual logger ui
             egui_logger::LoggerUi::default()
                 .enable_regex(true) // enables regex, default is true
+                .enable_search(true) // enables search bar, default is true
+                .max_log_length(2000) // sets maximum log messages to be retained, default is 1000
+                // Since we set "show_all_categories" to false in `main`, we should enable some
+                // categories to be shown by default.
+                .enable_category("hello".into(), true)
+                .enable_category("egui_glow::painter".into(), true)
```

Comment:
```suggestion
                .enable_category("egui_glow::painter".into(), true)
                // Since we only show explicitly enabled categories, this menu becomes useless
                .enable_categories_button(false) 

```

Reply:
Is this right, though? When testing this, the Category button does still seem to be useful. For example, when running the `hello.rs` example, I could enable the `eframe` category via the button to see some additional debug logs. The only thing I can't do anymore is disable categories which were explicitly enabled.

Reply:
> Is this right, though? When testing this, the Category button does still seem to be useful

right... my mistake. 

> The only thing I can't do anymore is disable categories which were explicitly enabled.

Do you see this as an Issue that needs to be fixed? I would be happy to merge this PR either way

Reply:
Hmm, that's a good question… I think it might be nice to still allow the user to disable such categories, but I'd probably add this in a follow-up issue (unless you'd like to have this implemented before the PR is merged).

Reply:
A follow up issue sounds good to me